src/cascadia/UnitTests_TerminalCore/ConptyRoundtripTests.cpp

```diff
@@ -2359,3 +2361,57 @@ void ConptyRoundtripTests::BreakLinesOnCursorMovement()
 
     verifyBuffer(termTb, term->_mutableViewport.ToInclusive());
 }
+
+void ConptyRoundtripTests::TestCursorInDeferredEOLPositionOnNewLineWithSpaces()
+{
+    auto& g = ServiceLocator::LocateGlobals();
+    auto& renderer = *g.pRender;
+    auto& gci = g.getConsoleInformation();
+    auto& si = gci.GetActiveOutputBuffer();
+    auto& hostSm = si.GetStateMachine();
+
+    auto& hostTb = si.GetTextBuffer();
+    auto& termTb = *term->_buffer;
+    const auto termView = term->GetViewport();
+
+    _flushFirstFrame();
+    _checkConptyOutput = false;
+
+    // newline down to the bottom
+    hostSm.ProcessString(std::wstring(gsl::narrow_cast<size_t>(TerminalViewHeight), L'\n'));
+    // fill width-1 with "A", then add one space and another character..
+    hostSm.ProcessString(std::wstring(gsl::narrow_cast<size_t>(TerminalViewWidth) - 1, L'A') + L" B");
+
+    auto verifyBuffer = [&](const TextBuffer& tb, SHORT bottomRow) {
+        // Buffer contents should look like the following: (80 wide)
+        // (w) means we hard wrapped the line
+        // (b) means the line is _not_ wrapped (it's broken, the default state.)
+        // cursor is on the '_'
+        //
+        // | ^ ^ ^ ^ ^ ^ ^ | ( ) (entire buffer above us; contents do not matter)
+        // |               | ( ) (entire buffer above us; contents do not matter)
+        // |AAAAAAAA...AAA | (w) (79 'A's, one space)
+        // |B_      ...    | (b) (There's only one 'B' on this line)
+

```

Comment:
Can you ASCII art what it looked like BEFORE too (above?)

Reply:
Sorry, I misread this test as adjusting the size after printing this in. There's no BEFORE/AFTER. Ignore.